deploy/production/etc/cron.d/physionet

```diff
@@ -16,3 +16,6 @@
 
 # auto reject pending credentialing applications in case the references don't respond
 0 */1 * * *  www-data  env DJANGO_SETTINGS_MODULE=physionet.settings.production /physionet/python-env/physionet/bin/python3 /physionet/physionet-build/physionet-django/manage.py reject_pending_credentialing_applications
+
+# auto remind users of pending credentialing applications in case the references don't respond(sent before the auto reject)
+0 */1 * * *  www-data  env DJANGO_SETTINGS_MODULE=physionet.settings.production /physionet/python-env/physionet/bin/python3 /physionet/physionet-build/physionet-django/manage.py remind_pending_credentialing_applications
```

Comment:
Instead of `remind_pending_credentialing_applications`, perhaps we should make it clear this relates to the reference? e.g. `remind_reference_identity_check`.

Reply:
Sounds good, i like the name `remind_reference_identity_check`. it much clearer.